test-config.edn

```diff
@@ -6,4 +6,6 @@
  :test-database-url "postgresql://localhost/rems_test?user=rems_test"
  :search-index-path "target/search-index-test"
  :authentication :fake-shibboleth
- :enable-pdf-api true}
+ :enable-pdf-api true
+ ;; list all organizations used in test-data
+ :organizations ["" "abc" "nbn" "organization1" "organization2"]}
```

Comment:
Hmm, is "perf" for performance data one of organizations that should be listed here?

Reply:
good catch, I double-checked the list against the test db